core/geometry/src/test/GeometryCoreTestIO.ts

```diff
@@ -35,7 +35,7 @@ export class GeometryCoreTestIO {
         fs.mkdirSync(path);
     }
     let fullPath = `${path}/${fileName}`;
-    if (fileName.search("\.imjs$") === 0)
+    if (fileName.search(".imjs$") === 0)
```

Comment:
Actually we do intend to search for the literal string ".imjs" at the end of `fileName`. So the "\\" is needed here.

However, we should be testing for -1 not 0. The intent is to ensure fullPath ends in ".imjs".

Can you make that change instead? :)